Gems/ROS2/Code/Source/RobotControl/Controllers/AckermannController/AckermannControlComponent.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "AckermannControlComponent.h"
+#include "ROS2/VehicleDynamics/VehicleInputControlBus.h"
```

Comment:
This include should use angle_brackets.
```suggestion
#include <ROS2/VehicleDynamics/VehicleInputControlBus.h>
```